backend/api/Controllers/Models/IsarMissionDefinition.cs

```diff
@@ -52,7 +52,7 @@ public IsarTaskDefinition(PlannedTask plannedTask, ITagPositioner tagPositioner)
                 .FirstOrDefault(t => t.TimeInSeconds.HasValue)
                 ?.TimeInSeconds;
 
-            Pose = tagPositioner.GetPoseFromTag(plannedTask.TagId);
+            Pose = new IsarPose(pose: plannedTask.Pose);
```

Comment:
```suggestion
            Pose = new IsarPose(plannedTask.Pose);
```
Consistency, named parameters are not normally used in C#